test-framework/test-suites/integration/tests/report/test_report_host_interface.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+@pytest.mark.usefixtures('revert_database')
```

Comment:
Chris has started harping on removing these since they're redundant, so I'm going to do that for him.